components/BundleModal.tsx

```diff
@@ -238,19 +241,23 @@ function BundleTransaction(transaction, index: number) {
       </td>
       <td className="flex flex-col px-6 py-4 whitespace-nowrap text-xs justify-center">
           {
-            Object.keys(coins).map(coin => <div className="flex flex-row items-center">
-              <a key={ "a_" + index + now() } className="flex hover:underline" target="_blank" rel="noreferrer" href={`https://etherscan.io/address/${ coins[coin].address }`} style={{ margin: 3}}>
-                {
-                  coins[coin].logo
-                    ? <img className="w-4 mr-1" key={ "i_" + index + now() } src={coins[coin].logo} />
-                    : <></>
-                }
-                <b>{ coin }</b>
-              </a>
-              { coins[coin].value > 0 ? " " + coins[coin].value : "" }
-              { coins[coin].ethValue > 0 ? " ($"+coins[coin].ethValue +")" : "" }
-              { coins[coin].value ? "" : ` (${coins[coin].event})` }
-            </div>)
+            Object.keys(coins).map(coin => 
+              <div className="flex flex-row items-center">
+                <a className="flex hover:underline min-w-max" target="_blank" rel="noreferrer" href={`https://etherscan.io/address/${ coins[coin].address }`} style={{ margin: 3}}>
+                  {
+                    coins[coin].logo
+                      ? <img className="w-4 mr-1" key={ "i_" + index + now() } src={coins[coin].logo} />
+                      : <></>
+                  }
+                  <b>{ coin }</b>
+                </a>
+                <span style={{ margin: 2}}>
+                  { coins[coin].value > 0 ? ` ${coins[coin].value}` : "" }
+                  { coins[coin].ethValue > 0 ? ` ($${coins[coin].ethValue})` : "" }
+                  { coins[coin].event ? ` (${coins[coin].event})` : ""  }
```

Comment:
As mentioned in https://github.com/martriay/flashbots-explorer/pull/17#discussion_r649367680,  please show event name only when no value has been transferred.